ipatests/test_integration/test_caless.py

```diff
@@ -1553,3 +1553,32 @@ def test_replica_ipa_ca_install(self):
 
         ca_replica = tasks.install_ca(self.replicas[0])
         assert ca_replica.returncode == 0
+
+
+class TestReplicaCALessToCAFull(CALessBase):
+    """
+    Test replica caless to cafull when master stays caless scenario:
+    Master (caless) / replica (caless) >> replica (ca)
+    """
+    num_replicas = 1
+
+    @classmethod
+    def install(cls, mh):
+        super(TestReplicaCALessToCAFull, cls).install(mh)
+
+        cls.create_pkcs12('ca1/server')
+        cls.prepare_cacert('ca1')
+        master = cls.install_server()
+        assert master.returncode == 0
+
+    def test_replica_ipa_ca_install(self):
+        """Install CA on replica (master caless)"""
```

Comment:
This comment is inaccurate, the test actually install CA-less replica and then installs CA on the replica.